src/Components/Form/MultiSelectMenuV2.tsx

```diff
@@ -177,7 +178,7 @@ export default MultiSelectMenuV2;
 
 interface MultiSelectOptionChipProps {
   label: ReactNode;
-  onRemove?: () => void;
+  onRemove?: (e: any) => void;
```

Comment:
```suggestion
  onRemove?: (e: React.MouseEvent<HTMLParagraphElement>) => void;
```

@UdaySagar-Git try to avoid `any` and `unknown` as much as possible. Also just apply this change and check if there any type error or a typo (just in case 😅 )

Reply:
I'd suggest do the `e.stopPropogation` in the multi select option chip component itself as as per the props type, argument `e` is not passed.

<img width="1124" alt="image" src="https://github.com/coronasafe/care_fe/assets/25143503/4e2deb25-8afd-4d26-9c17-f1d4602c0bb9">

Reply:
sure